controller/sp-validator/tmpl/validating_webhook_configuration.go

```diff
@@ -0,0 +1,22 @@
+package tmpl
+
+// ValidatingWebhookConfigurationSpec provides a template for a
+// ValidatingWebhookConfiguration.
```

Comment:
yes we will do that separately alongside with the auto-injector's in #2176. No trivial because it requires the generation of the CA Bundle (or accept it from the user as a CLI option).